packages/ui/src/scss/modal.module.scss

```diff
@@ -7,8 +7,38 @@
   outline: none;
 }
 
+.drawer {
+  position: fixed;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  max-height: 411px;
+  outline: none;
+  padding-bottom: env(safe-area-inset-bottom);
+}
+
 .overlay {
   position: fixed;
   background: rgba(0, 0, 0, 16%);
   inset: 0;
 }
+
+
+// React modal animation
+/* stylelint-disable-next-line selector-pseudo-class-no-unknown */
+:global {
+  .ReactModal__Content[role='popup'] {
+    transform: translateY(100%);
+    transition: transform 300ms ease-in-out;
```

Comment:
nit: We usually use `0.3` instead of `300ms`

Reply:
yep, was misled by the stylelint . updated